scripts/ansible/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 cryptography==3.3.2
-ansible==2.8.8
+ansible==2.9.9
```

Comment:
Any specific reason for updating to 2.9.9 rather than the latest, 2.9.18?

Reply:
No specific reason, it might have been the first version with the fix for 20.04. I'll update to .18